packages/lib/src/vis/shared/SelectionRect.tsx

```diff
@@ -1,23 +1,44 @@
 import type { Color } from '@react-three/fiber';
+import { useMemo } from 'react';
 import type { Vector2 } from 'three';
+import { PlaneGeometry } from 'three';
 
 interface Props {
   startPoint: Vector2;
   endPoint: Vector2;
   color?: Color;
   opacity?: number;
+  borderColor?: Color;
 }
 
 function SelectionRect(props: Props) {
-  const { startPoint, endPoint, color = 'red', opacity = 0.5 } = props;
+  const {
+    startPoint,
+    endPoint,
+    color = 'red',
+    opacity = 0.5,
+    borderColor,
+  } = props;
```

Comment:
`SelectionRect` is now a bit more generic: it can display a filled rectangle **and** a border. By default, `borderColor` is `undefined` which disable the border.